pkg/ds/daemon_set.go

```diff
@@ -45,6 +48,8 @@ type DaemonSet interface {
 	// Returns a list of all nodes that are selected by this daemon set's selector
 	EligibleNodes() ([]types.NodeName, error)
 
+	metricName(name string) string
```

Comment:
can we lowercase the interface name to make that clearer? or is it referred to anywhere